events/schema.py

```diff
@@ -146,6 +147,8 @@ def resolve_type(cls, instance, info):
             return TicketmasterEventTicketSystem
         elif instance.ticket_system == Event.LIPPUPISTE:
             return LippupisteEventTicketSystem
+        elif instance.ticket_system == Event.TIXLY:
+            return TixlyEventTicketSystem
```

Comment:
As this is the fourth if-else clause some refactoring would be nice, not necessary in this PR though, e.g.:
```python
EVENT_TICKET_SYSTEM_TYPES = {
    Event.INTERNAL: InternalEventTicketSystem,
    Event.TICKETMASTER: TicketmasterEventTicketSystem,
    Event.LIPPUPISTE: LippupisteEventTicketSystem,
    Event.TIXLY: TixlyEventTicketSystem,
}
...
    @classmethod
    def resolve_type(cls, instance, info):
        if instance.ticket_system in EVENT_TICKET_SYSTEM_TYPES:
            return EVENT_TICKET_SYSTEM_TYPES[instance.ticket_system]
        raise Exception(f'Invalid ticket system "{instance.ticket_system}".')
```

Reply:
Yes, I do agree, but also, the classes are actually introduced later in the code and a try except would make the implementation as long as it currently is. I would do it later (or next time), since we got quite much to do at the moment.